ee/codegen/src/context/node-context/base.ts

```diff
@@ -96,7 +97,12 @@ export abstract class BaseNodeContext<T extends WorkflowDataNode> {
       bases: [
         {
           name: this.baseNodeClassName,
-          module: [...this.baseNodeClassModulePath],
+          module: [
+            ...(this.isCore
+              ? this.workflowContext.sdkModulePathNames.CORE_NODES_MODULE_PATH
+              : this.workflowContext.sdkModulePathNames
+                  .DISPLAYABLE_NODES_MODULE_PATH),
+          ],
```

Comment:
`this.baseNodeClassModulePath` is used elsewhere for imports of the base class. I imagine we still want _those_ to say displayable, but for node definitions, we need the exact path of where it's defined. Open to other ideas of organizing this

Reply:
Seems reasonable. If there's reason to change later, we can